clang/lib/DPCT/AnalysisInfo.h

```diff
@@ -180,7 +180,7 @@ struct CudaArchPPInfo {
   std::unordered_map<unsigned, DirectiveInfo> ElInfo;
   bool isInHDFunc = false;
 };
-
+enum class CodePinVarInfoType { Field, Base, Alias };
```

Comment:
it is not straightforward to understand the 3 key words, could you add comment for them?